driver/fuzz_target_runner.cpp

```diff
@@ -78,7 +81,11 @@ FuzzTargetRunner::FuzzTargetRunner(
     JVM &jvm, const std::vector<std::string> &additional_target_args)
     : ExceptionPrinter(jvm), jvm_(jvm), ignore_tokens_() {
   auto &env = jvm.GetEnv();
-  // the first positional argument should be the fuzz target class
+  if (FLAGS_target_class.empty()) {
+    FLAGS_target_class = DetectFuzzTargetClass();
+  }
+  // If automatically detecting the fuzz target class failed, we expect it as
```

Comment:
The command was inaccurate before as well, maybe we can fix it now: the fuzz target class is not passed as a positional argument.